spec/acceptance/replset_spec.rb

```diff
@@ -201,30 +200,29 @@ class { 'mongodb::globals':
         nc1ohyB0lNt8lHf1U00mtgDSV3fwo5LkwhRi6d+bDBTL/C6MZETMLdyCqDlTdUWG
         YXIsJ0gYcu9XG3mx10LbdPJvxSMg',
         }
-        -> class { 'mongodb::client': }
+        -> mongodb_replset { 'test':
+          members => [#{hosts.map { |x| "'#{x}:27017'" }.join(',')}],
+        }
+
+        unless $facts['mongodb_is_master'] == 'false' { # lint:ignore:quoted_booleans
```

Comment:
Copied this from the mongodb:db define to make sure the user is only managed on the primary.